code/game/objects/effects/contraband.dm

```diff
@@ -546,7 +546,7 @@
 
 /obj/structure/sign/poster/official/anniversary_vintage_reprint
 	name = "50th Anniversary Vintage Reprint"
-	desc = "A reprint of a poster from 2505, commemorating the 50th Anniversary of Nanoposters Manufacturing, a subsidiary of Nanotrasen."
+	desc = "A reprint of a poster from 2505, commemorating the 50th Anniversary of Nanoposters Manufacturing."
```

Comment:
Are we still keeping `Nanoposters`?

Reply:
They're made with Nanotechnology! Unless we want to shift away from it, I don't see why not.